xml/System.Numerics/BigInteger.xml

```diff
@@ -1545,9 +1545,9 @@
         <Parameter Name="isUnsigned" Type="System.Boolean" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="isUnsigned">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="isUnsigned"><see langword="true" /> to use unsigned encoding; otherwise, <see langword="false" />.</param>
```

Comment:
Why are all the params first?

Reply:
Not sure what do you mean. Can you elaborate?

Reply:
Not sure what you mean. Can you please elaborate?

Reply:
In xml comments, `summary` is generally at the top; followed by `params`, `return`, and `remarks`.

I was asking why `param` is appearing before `summary` for all of these.

Reply:
Ah no, in dotnet-api-docs the structure is slightly different. In fact, the xml file "skeleton" is automatically generated as you see it.
In the end it doesn't matter, because what matters is how it's all shown in the MS Docs website.